revscoring/features/wikitext/datasources/parsed.py

```diff
@@ -190,6 +206,20 @@ def tag_names_matching(self, regex, name=None):
 
         return filters.regex_matching(regex, self.tag_names, name=name)
 
+    def tags_str_with_name_matching(self, regex, name=None):
+        """
+        Constructs a :class:`revscoring.Datasource` that returns all tags having a name
+        that matches a regular expression as strings.
+        """
+        if not hasattr(regex, "pattern"):
+            regex = "<{}.*>".format(regex)
```

Comment:
I'm not sure I like this strategy.  I think we should either allow the regex match the complete tag and content or we should use mwparserfromhell to extract the tag name to match against.